docs/Revocation.md

```diff
@@ -0,0 +1,96 @@
+# Revocation 
+
+As of version `1.6.0` The Inji verify supports verification of `Revoked` vc. Revocation is a mechanism used by an issuer to express the status of a Claim after issuance. 
+
+# Why is VC Revocation needed?
+
+In a `Verifiable Credential` (VC), once it’s issued, it’s cryptographically signed and can be verified even offline.
+
+But what happens if the credential later becomes invalid? For example:
+- A driver’s license is revoked
+- A student’s degree is withdrawn
+- An employment certificate is expired or suspended
+
+We need a way to check if the VC is still valid — without reissuing it.
+That’s where the `credentialStatus` property comes in.
+It points to a `status list` or registry maintained by the issuer, where verifiers can check whether the credential has been `revoked`, `suspended`, or otherwise invalidated.
+
+## API Documentation
+
+The API documentations can be found in the [Inji Verify API documentation](https://mosip.stoplight.io/docs/inji-verify/branches/main/)
+
+## Functionalities 
+
+## 1. UPLOAD & SCAN
+
+### Verifiable Credential Submission:
+- Inji Verify have API to submit Verifiable Credential.
+
+> **Note** that Verifiable Credential Submission endpoint(`/vc-submission`) is specifically for submitting individual credentials and is distinct from a Verifiable Presentation (VP) submission.
+
+- Once the verifier scans the QR code, it decodes the QR code using `pixel_pass_library` and post it to the Inji Verify backend (db).
+
+- A typical JSON-LD Revoked Verifiable Credential includes a field like this:
+
+    ```json
+    "credentialStatus": {
+    "id": "https://example.org/status/24#94567",
+    "type": "StatusList2021Entry",
+    "statusPurpose": "revocation",
+    "statusListIndex": "94567",
+    "statusListCredential": "https://example.org/status/24"
+    }
+    ```
+
+- Field-by-Field Explanation
+
+| Field                 | Meaning                                                                                                   |
+|-----------------------|-----------------------------------------------------------------------------------------------------------|
+| id                    | A unique URL identifier for this status entry.                                                             |
+| type                  | The method used for status checking — e.g., `BitstringStatusListEntry`.                                   |
+| statusPurpose         | Describes why this status exists — e.g., "revocation" or "suspension".                                     |
+| statusListIndex       | A numeric index (or bit position) representing this credential’s position in the status list.             |
+| statusListCredential  | The URI of the status list credential — a VC published by the issuer that contains a compressed bitstring. |
+
+### Submission Result:
+
+- Inji verify UI can fetch the result of the submission through APIs.
```

Comment:
The current note may confuse readers because it mentions the `/vc-submission` endpoint in Line R29 but doesn’t clarify that the Verify UI does not use this endpoint at all.
Please update the documentation to make it clear that:

Results submitted to `/vc-submission` are not fetched by the Verify UI.

The Verify UI only fetches results submitted to the `vp-submission/direct-post` endpoint.

This clarification will help avoid misunderstanding between VC submission and VP submission flows.